scripts/serverless.js

```diff
@@ -514,6 +514,7 @@ const processSpanPromise = (async () => {
     const isStandaloneCommand = notIntegratedCommands.has(command);
 
     if (!isHelpRequest && (isInteractiveSetup || isStandaloneCommand)) {
+      let interactiveContext;
```

Comment:
It'll be nicer to reuse the top level `serverless` variable (same as we reuse `configuration` one).